File: 100-Days-Series/Day 57/NumberPyramid.cpp
#include<iostream>
using namespace std;
// Function to print Number Pyramid
void getPrintPyramid(int n){
    for(int i=1;i<=n;i++){
        // Print Spaces
        for(int j=i;j<n;j++){
            cout<<" ";
        }
        // Print Ascending numbers
        for(int j=1;j<=i;j++){
            cout<<j;
        }
        // Print Descending numbers
        for(int j=i-1;j>=1;j--){
            cout<<j;
        }
        cout<<endl;
    }
}
// Let's Start here
int main(){
    int rows;
    cout<<"\nEnter the number of rows : ";
    cin>>rows;
    getPrintPyramid(rows);
}

File: 100-Days-Series/Day 57/NumberPyramid.java

public class NumberPyramid {
    // Functionn to prnit Number Pyramid
    public static void getPrintPyramid(int n){
        for(int i=1;i<=n;i++){
            // Print Spaces
            for(int j=i;j<n;j++){
                System.out.print(" ");
            }
            // print ascending numbers
            for(int j=1;j<=i;j++){
                System.out.print(j);
            }
            // print descending numbers
            for(int j=i-1;j>=1;j--){
                System.out.print(j);
            }
            System.out.println();
        }
    }
    public static void main(String [] args){
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of rows : ");
        int rows=sc.nextInt();
        getPrintPyramid(rows);
    }
}
